packages/lit-element/src/lit-element.ts

```diff
@@ -97,15 +97,17 @@ let issueWarning: (code: string, warning: string) => void;
 if (DEV_MODE) {
   // Ensure warnings are issued only 1x, even if multiple versions of Lit
   // are loaded.
-  const issuedWarnings: Set<string | undefined> =
-    (globalThis.litIssuedWarnings ??= new Set());
+  globalThis.litIssuedWarnings ??= new Set();
```

Comment:
some of your tests are re-assigning `globalThis.litIssuedWarnings`.
to better support that use case, rather than having local `issuedWarnings` variable, always check what is the current globalThis.litIssuedWarnings value